File: BuddhaGameServer/src/log/outputs/impls/SystemLogOutput.java
package log.outputs.impls;

import log.Log;
import log.outputs.LogOutput;

public class SystemLogOutput extends LogOutput {
    @Override
    public void write(int level, String message) {
        if(level >= Log.sOutputLevel) {
            if (level == 2) {
                System.err.println(message);
            } else {
                System.out.println(message);
            }
        }
    }
}

File: BuddhaGameClient/Assets/Scripts/Utils/Log/Outputs/Impls/SystemLogOutput.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

namespace BuddhaGame
{
    public class SystemLogOutput : LogOutput
    {
        public override void write(int level, string message)
        {
            if (level >= Log.sOutputLevel)
            {
                if (level == 2)
                {
                    Debug.LogError(message);
                }
                else
                {
                    Debug.Log(message);
                }
            };
        }
    }
}
